src/Uno.UI/UI/Xaml/DependencyProperty.Dictionary.TypeNullableDictionary.cs

```diff
@@ -0,0 +1,47 @@
+﻿#nullable enable
+
+using System;
+using Uno.Collections;
+using Uno.UI.Helpers;
+using Uno.Extensions;
+
+namespace Windows.UI.Xaml
+{
+	public sealed partial class DependencyProperty
+	{
+		private readonly static TypeNullableDictionary _isTypeNullableDictionary = new TypeNullableDictionary();
+
+		private bool BuildIsTypeNullable(Type type)
```

Comment:
Not sure about the `Build` prefix ... it's a cached value ^^
```suggestion
		private bool IsTypeNullable(Type type)
```